.gitlab-ci.yml

```diff
@@ -1,29 +1,18 @@
-image: openjdk:11
-
-workflow:
-  rules:
-    - if: $CI_EXTERNAL_PULL_REQUEST_TARGET_BRANCH_NAME == 'dev'
-
-stages:
-  - build
-  - test
+include:
+  remote: 'https://raw.githubusercontent.com/ebi-ait/gitlab-ci-templates/master/build-release-deploy.yml'
```

Comment:
The GitLab setup should also be documented somewhere. Might be good to put in the same repo as this template!

Reply:
Yes! Will do

Reply:
Done